core/connection/manager.go

```diff
@@ -47,12 +48,7 @@ var (
 )
 
 // Creator creates new connection by given options and uses state channel to report state changes
-// Given options:
-//  - session,
-//  - consumer identity
-//  - service provider identity
-//  - service proposal
-type Creator func(ConnectOptions, StateChannel, StatisticsChannel) (Connection, error)
+type Creator func(string, StateChannel, StatisticsChannel) (Connection, error)
```

Comment:
Can we name arguments to explain what should be passed? Just `string` says nothing what should be there.